lighthouse-core/gather/driver.js

```diff
@@ -150,7 +150,11 @@ class Driver {
         returnByValue: true
       }).then(result => {
         clearTimeout(asyncTimeout);
-        resolve(result.result.value);
```

Comment:
nice catch! I'm wondering if we should move this into `Connection` itself though. We already [check for an `error` property on the result](https://github.com/GoogleChrome/lighthouse/blob/0220f269a23bb20917b0a5b4e6dfedf0d5fb95ca/lighthouse-core/gather/connections/connection.js#L98), so should we be checking for `exceptionDetails` and rejecting for that as well?